yt_dlp/extractor/franceculture.py

```diff
@@ -67,7 +102,6 @@ def _real_extract(self, url):
             'thumbnail': thumbnail,
             'ext': ext,
             'vcodec': 'none' if ext == 'mp3' else None,
-            'uploader': uploader,
-            'timestamp': int_or_none(video_data.get('data-start-time')) or int_or_none(video_data.get('data-asset-created-date')),
+            'upload_date': upload_date.replace('-', '') if upload_date is not None else None,
```

Comment:
Use `unified_strdate`

and why remove uploader and timestamp?

Reply:
Done thanks.